.eslintrc.js

```diff
@@ -4,7 +4,9 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0
-  },
+    'no-proto': 0,
+   'comma-dangle': 0,
```

Comment:
Disabling 'comma-dangle' might lead to inconsistent code style. Consider setting it to 'always-multiline' or 'never' to maintain consistency.